site/data/refDataApi.ts

```diff
@@ -151,13 +151,18 @@ export const fetchServiceStops = async (input: FetchServiceStops) => {
 
 interface FetchOperatorsInput {
     adminAreaCodes: string[];
+    dataSource?: Modes;
 }
 
 export const fetchOperators = async (input: FetchOperatorsInput) => {
     const searchApiUrl = `${API_BASE_URL}/operators`;
 
     const queryStringItems = [`adminAreaCodes=${input.adminAreaCodes.join(",")}`];
 
+    if (input.dataSource) {
+        queryStringItems.push(`dataSource=${input.dataSource}`);
```

Comment:
why not just add it to the array on line 160 const queryStringItems = [`adminAreaCodes=${input.adminAreaCodes.join(",")}`,`dataSource=${input.dataSource}` ];

Reply:
Its better to add the datasource is it is passed as input